src/plan_add_hashagg.c

```diff
@@ -24,6 +25,8 @@
 #include "guc.h"
 #include "estimate.h"
 
+#define GAPFILL_PATH_NAME "GapFill"
```

Comment:
Can this constant be shared with Custom Node creation and used there too?

Reply:
ideally yes but that requires some more refactoring because gapfill is tsl and we need this check to be in apache code